torchaudio/functional/functional.py

```diff
@@ -1444,3 +1446,82 @@ def resample(
                                               resampling_method, beta, waveform.device, waveform.dtype)
     resampled = _apply_sinc_resample_kernel(waveform, orig_freq, new_freq, gcd, kernel, width)
     return resampled
+
+
+def character_edit_distance(r: str, h: str) -> int:
+    """
+    Calculate the character level edit (Levenshtein) distance between two strings.
+
+    The function computes an edit distance allowing deletion, insertion and substitution.
+    The result is an integer. Users may want to normalize by the length of the reference.
+
+    This can be used to compute the edit distance for instance between two strings,
```

Comment:
Is this statement still true now that it's split into two functions? This flexibility also makes it inconsistent with the function typing and documentation

Reply:
I think this is still true as now we are just splitting up charter and word level edit distance. You may delete, insert or substitute  a charter or a word at a time.

Reply:
I think what @carolineechen meant is that the following part is inconsistent with what type annotation / docstring says.

```
This can be used ... or two list of words. Note that, if a string and a list of words is provided, the distance
     will be computed between the "string" sequence, and the "list of words" sequence.
```

Reply:
Thanks for pointing it out. I've removed these comments :)